File: ios/RCTMGL-v10/RCTMGLLineLayerManager.m
#import <React/RCTBridgeModule.h>
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_MODULE(RCTMGLLineLayerManager, RCTViewManager)

// circle layer props
RCT_EXPORT_VIEW_PROPERTY(sourceLayerID, NSString)

// standard layer props
#include "CommonLayerProperties.H"

@end

File: ios/RCTMGL-v10/RCTMGLLineLayerManager.swift
@objc(RCTMGLLineLayerManager)
class RCTMGLLineLayerManager: RCTViewManager {
    @objc
    override static func requiresMainQueueSetup() -> Bool {
        return true
    }
  
    override func view() -> UIView! {
      let layer = RCTMGLLineLayer()
      layer.bridge = self.bridge
      return layer
    }
}
